frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -523,7 +523,9 @@ ACTION_TYPE_FAILED_STORAGE_CONNECTION_FOR_DOMAIN_ALREADY_EXISTS=Cannot ${action}
 ACTION_TYPE_FAILED_STORAGE_CONNECTION_FOR_DOMAIN_NOT_EXIST=Cannot ${action} ${type}. Storage connection is not attached to the specified domain.
 ACTION_TYPE_FAILED_STORAGE_CONNECTION_ID_EMPTY=Cannot ${action} ${type}. Storage connection id is empty.
 ACTION_TYPE_FAILED_STORAGE_CONNECTION_NOT_EXIST=Cannot ${action} ${type}. Storage connection doesn't exist.
-ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FOR_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type}. Managed Block Storage is not supported this operation.
+ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FOR_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type}. Managed Block Storage does not support this operation.
+ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FROM_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type} from Managed Block Storage. Managed Block Storage does not support this operation.
+ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_TO_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type} to Managed Block Storage. Managed Block Storage does support this operation.
```

Comment:
@ArtiomDivak those message don't reflect the situation good enough - they need to indicate that the operation is forbidden because we do it from/to a local SD
maybe we should just say that copy to/from local SD is forbidden?
@bennyz ^

Reply:
> @ArtiomDivak those message don't reflect the situation good enough - they need to indicate that the operation is forbidden because we do it from/to a local SD maybe we should just say that copy to/from local SD is forbidden? @bennyz ^

I agree

Reply:
ACK

Reply:
I changed the message is it ok now?